src/logic/parsers/threephase/threephaseparser.py

```diff
@@ -167,160 +174,163 @@ def get_load_connection(self, model, simulation_state, phase):
 
         return bus
 
-    def create_capacitors(self, simulation_state: DxNetworkModel):
-        for model in self.ditto_store.models:
-            if isinstance(model, ditto.models.capacitor.Capacitor):
-                gld_cap = self.all_gld_objects[model.name]
-                for phase_capacitor in model.phase_capacitors:
-                    if not phase_capacitor.phase in model.connected_phases:
-                        continue
+    def get_load_name(self, model: ditto.models.load.Load, phase_load):
+        load_num = model.name.split("_")[-1]
+        if phase_load.phase.isalpha():
+            load_name = f"cl_{load_num}{phase_load.phase}"
+        elif phase_load.phase.isnumeric():
+            load_name = f"rl_{load_num}_{phase_load.phase[0]}"
+        else:
+            load_name = f"{load_num}_{phase_load.phase}"
+        return load_name
```

Comment:
You know, if we're going to have some kind of canonical name for network elements, maybe this should exist as a field on the model itself? That way it's easier to get at.

Reply:
I don't particularly like this naming scheme (it uses "rl" for residential load and "cl" for commercial load which I don't find intuitive, and it denotes both phase 1 and phase 12 triplex loads as just "_1"). but it exists on the specific load files I'm making use of from Hoke et al. I could rename this method something like "get_hoke_et_al_load_name" to indicate its specific usage.

Reply:
ah, I missed that this was to map back to the time series data. In that case, could this live in the post processor since it already 'knows' about the csv format?